contracts/leaser/src/msg.rs

```diff
@@ -71,6 +72,7 @@ pub enum SudoMsg {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum QueryMsg {
+    #[cfg(any(test, feature = "contract"))]
```

Comment:
do not have time to dig into this change ... could you elaborate more on the motivation od doing it?

Reply:
It tries to expose code which is feature gated. Variants were to either lock `msg` module under the `contract` feature, which is an overkill, or to just filter out the parts which required it.

Reply:
in the spirit of the previous my comment this should be reverted as well